MaxText/configs/base.yml

```diff
@@ -63,15 +63,20 @@ int8_training: False
 # base_mlp_dim, base_num_decoder_layers and/or head_dim.
 global_parameter_scale: 1
 base_emb_dim: 2560
+base_num_heads: 8
 base_num_query_heads: 8
 base_num_kv_heads: 8
 base_mlp_dim: 8192
 base_num_decoder_layers: 16
 head_dim: 256
+
```

Comment:
Nit: please avoid random changes. (If this change is deliberate, it is ok.)

Reply:
Gotcha, will take care.